### PR TITLE
add material card for persons

### PR DESCRIPTION
**Pull request by TomGallon**
add material card for persons

### CODE REVIEW

File: app/components/material-card.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                {actions}
+            </div>
+        </div>
+    );
+}
+
+MaterialCard.displayName = 'MaterialCard';
+MaterialCard.PropTypes = {
+    imgUrl: PropTypes.string,
+    core: PropTypes.element,
```
Comment (pierr) [inline app/components/material-card.jsx]: I would have name it supporting text

**Comment by pierr** [comment]
It is not really a problem but the card is splitter in many components.

File: app/components/material-card.jsx (hunk 1 above)
Comment (TomGallon) [inline app/components/material-card.jsx]: Ok, maybe. 
But this class part of the mdl Card template. 
I can rewrite a new one in css but no added value...

File: app/styles/_mixins.scss [hunk 2]
```diff
@@ -6,6 +7,59 @@
...
+    height: 440px;
+    & > div {
+        @include background-cover($defaultImgUrl);
+        width: 100%;
+        height: 100%;
+        overflow: hidden;
+        img {
+            width: 100%;
+            filter: grayscale($black-and-white-variant);
+            -webkit-filter: grayscale($black-and-white-variant);
```
Comment (pierr) [inline app/styles/_mixins.scss:22]: sass mixin greyscale

**Pull request merged**